src/AVFoundation/AVTypes.cs

```diff
@@ -266,4 +266,39 @@ public static RectangleF WithAspectRatio (this RectangleF self, SizeF aspectRati
 		}
 	}
 #endif
+
+	[Mac (10, 10), NoiOS, NoWatch, NoTV]
+	[StructLayout (LayoutKind.Sequential)]
+	public struct AVSampleCursorSyncInfo {
+		public bool SampleIsFullSync;
```

Comment:
Not sure we need the `Sample` prefix on every fields. Check if they exists in swift.